x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -116,19 +123,19 @@ structures:
 
 == About Areas Key
 
-In the previous schema example, the areas key has a value array of individual area definitions. Areas are similar to the breakdown of normal X12 transaction sets into header, detail, and summary sections, but provide much finer granularity.  Instead of the three fixed portions of a transaction set defined in X12, there may be twenty or more areas defined in a HIPAA transaction set.
+In the previous schema example, the areas key has a value array of individual area definitions. Areas are similar to the breakdown of normal X12 transaction sets, for example into header, detail, and summary sections, but provide much finer granularity.  Instead of the three fixed portions of a transaction set as defined in X12 there may be twenty or more areas defined in a HIPAA transaction set.
```

Comment:
I would put commas around "as defined in X12."

Reply:
Fixed.